pomodoro.sh

```diff
@@ -172,6 +206,82 @@ EOF
     eval "$1=$should_continue" # set first parameter to have the return type
 }
 
+############################################
+# TODO: adding better output 
+# ###########################################
+#
+STATIC_OUTPUT=""
+DYNAMIC_OUTPUT=""
+
+refresh_screen() {
+    # I'm having a hard time figuring out how to manage the output of single pomodoros even when the screen changes
+    # so I'll maintain a global list of what needs to change and add to it as pomodoros complete
+    # I'll then use this to refresh the whole screen and call this on each action I get and work on
+    tput clear # clear the whole screen
+    printf "%b" "$STATIC_OUTPUT"
+    printf "%b" "$DYNAMIC_OUTPUT"
+}
+
+
+refresh_current_pomodoro_output () {
+    # set -x
```

Comment:
remove debugging information and add comment in README on how to debug bash functions